test/functional/rpc_net.py

```diff
@@ -262,6 +263,12 @@ def test_addpeeraddress(self):
         assert_equal(node.addpeeraddress(address="1.2.3.4", port=8333), {"success": False})
         assert_equal(len(node.getnodeaddresses(count=0)), 1)
 
+        self.log.debug('Test adding a second address to the tried table')
+        assert_equal(node.addpeeraddress(address="1.2.3.0", port=8333, tried=True), {"success": True})
+        with node.assert_debug_log(expected_msgs=["new 1, tried 1, total 2"]):
```

Comment:
nit in the second commit:

Maybe assert on the longer string "Addrman checks started: new 1, tried 1, total 2" to clarify that this is triggered by the addrman check? Either in addition or as replacement for the comment below, which is not verified by the test itself.

Reply:
I held off because that aspect of the logging may change in #22872, but that pull can update the test too, so ok taking your feedback.

Reply:
done